vagrant/config/versions.yaml

```diff
@@ -87,6 +87,18 @@ installers:
       - 'debian10'
       - 'ubuntu1804'
 
+  - foreman: '2.3'
+    katello: 'nightly'
+    pulp: '2.21'
+    pulpcore: '3.6'
```

Comment:
Not sure, but I think they want 3.7. We can fix that later since nightly is also still on 3.6.

Reply:
+1 to fix it after nightly.

Reply:
@jlsherrill FYI